src/Microsoft.XmlSerializer.Generator/src/Sgen.cs

```diff
@@ -404,16 +404,16 @@ private void WriteHeader()
         private void WriteHelp()
         {
             Console.Out.WriteLine(SR.Format(SR.HelpDescription));
-            Console.Out.WriteLine(SR.Format(SR.HelpUsage, this.GetType().Assembly.GetName().Name));
+            Console.Out.WriteLine(SR.Format(SR.HelpUsage, this.GetType().Assembly.GetName().Name.Substring(7)));
             Console.Out.WriteLine(SR.Format(SR.HelpDevOptions));
-            Console.Out.WriteLine(SR.Format(SR.HelpAssembly, "/assembly:", "/a:"));
-            Console.Out.WriteLine(SR.Format(SR.HelpType, "/type:", "/t:"));
-            Console.Out.WriteLine(SR.Format(SR.HelpProxy, "/proxytypes", "/p"));
-            Console.Out.WriteLine(SR.Format(SR.HelpForce, "/force", "/f"));
-            Console.Out.WriteLine(SR.Format(SR.HelpOut, "/out:", "/o:"));
+            Console.Out.WriteLine(SR.Format(SR.HelpAssembly, "--assembly:", "--a:"));
```

Comment:
For the short format, it should be `-a`. This needs to be changed everywhere else as well.